routes/web.php

```diff
@@ -92,4 +92,12 @@
      * Support Staffs
      */
     Route::resource('/support-staffs', 'SupportStaffController');
+
+    /**
+     * Chart of accounts
+     */
+    Route::resource('/chart-of-accounts', 'GeneralLedgerAccountsController');
+    Route::get('/chart-of-accounts/{general_ledger_accounts}/add', 'GeneralLedgerAccountsController@addNewAccount');
+    Route::post('/chart-of-accounts/{general_ledger_accounts}/save', 'GeneralLedgerAccountsController@saveNewAccount');
```

Comment:
if you can merge these in the create and store functions then we won't need these extra routes

Reply:
Explained ☝️

Reply:
Explained ☝️